from lib.utils import run_problem
from codeforces.c641d2 import a


def test_a():
    run_problem(a.main, 'a')

